scripts-available/CDB_Overviews.sql

```diff
@@ -0,0 +1,220 @@
+CREATE OR REPLACE FUNCTION _CDB_Feature_Density(reloid REGCLASS, nz integer)
```

Comment:
So help me out here. This is exploratory, looking for the zoom level at which the feature density on a "typical" tile exceeds some threshold?